ext/nnstreamer/tensor_converter/tensor_converter_flexbuf.cc

```diff
@@ -124,12 +126,14 @@ flxc_convert (GstBuffer *in_buf, GstTensorsConfig *config, void *priv_data)
     flexbuffers::String _name = tensor[0].AsString ();
     const gchar *name = _name.c_str ();
 
-    config->info.info[i].name = (name && strlen (name) > 0) ? g_strdup (name) : NULL;
-    config->info.info[i].type = (tensor_type) tensor[1].AsInt32 ();
+    _info = gst_tensors_info_get_nth_info (&config->info, i);
+
+    _info->name = (name && strlen (name) > 0) ? g_strdup (name) : NULL;
```

Comment:
With g_strdup, you may incur a memory leak here as well.

Reply:
Same here. Let's call g_free, instead of gst_tensors_config_init (which can be duplicated in the context of gst-converter)